CMakeLists.txt

```diff
@@ -33,31 +33,57 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake)
 
+if(WIN32)
+  set(WIN_EXTRA_LIBS_PATH "C:/Programs" CACHE PATH
+    "Base path to extra libs/headers on Windows (SDL, dirent, pthreads, msinttypes folders should be in here).")
+  list(APPEND CMAKE_PREFIX_PATH "${WIN_EXTRA_LIBS_PATH}" "${WIN_EXTRA_LIBS_PATH}/SDL")  # hints for FindSDL
+endif()
+
 find_package(Qt5Widgets)
 find_package(Qt5Xml)
 find_package(Qt5LinguistTools)
 find_package(Qt5PrintSupport)
 find_package(Qt5Multimedia)
 
-message(STATUS "Looking for SDL ")
-find_package(Sdl)
```

Comment:
at the time we introduced SDL I think this find was missing, thanks for the change!